addons/l10n_account_withholding/wizards/account_payment_register.py

```diff
@@ -0,0 +1,275 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import json
+from collections import defaultdict
+
+from odoo import Command, api, fields, models
+from odoo.exceptions import UserError
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = 'account.payment.register'
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    display_withholding = fields.Boolean(compute='_compute_display_withholding')
+    withhold_tax = fields.Boolean(
+        string='Withhold Tax Amounts',
+        compute='_compute_withhold_tax',
```

Comment:
Not very clear... The name looks like a relational field, the string looks like a monetary field but... it's a boolean :rofl: